plotly-behaviours.html

```diff
@@ -0,0 +1,244 @@
+<link rel="import" href="plotly-import.html">
+<script>
+    window.PlotlyBehaviours = window.PlotlyPlotlyBehaviours || {};
```

Comment:
This is incorrect. Also, global variables are a bad idea for module patterning. I would suggest something like UMD if you actually want this to work with how people use it.